daemons/gptp/common/ptp_message.cpp

```diff
@@ -959,9 +962,8 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 	master_local_freq_offset += 1.0;
```

Comment:
Duh
2 ^ 41 != 2 << 41
eg
2 ^ 1 = 2
2 << 1 = 4
etc.
I get it now. Ignore my previous comment.